docs/source/es/_toctree.yml

```diff
@@ -64,8 +64,6 @@
     title: Entrenador
   - local: sagemaker
     title: Ejecutar el entrenamiento en Amazon SageMaker
-  - local: serialization
```

Comment:
in `es`, `it`, and `pt`: `serialization.md` was very outdated and doesn't match our recent practices -> delete